pkg/server/render.go

```diff
@@ -60,9 +61,26 @@ func (ctrl *Controller) renderHandler(w http.ResponseWriter, r *http.Request) {
 		out = &storage.GetOutput{Tree: tree.New()}
 	}
 
-	fs := out.Tree.FlamebearerStruct(p.maxNodes)
-	res := renderResponse(fs, out)
-	ctrl.writeResponseJSON(w, res)
+	switch p.format {
+	case "json":
+		fs := out.Tree.FlamebearerStruct(p.maxNodes)
+		res := renderResponse(fs, out)
+		ctrl.writeResponseJSON(w, res)
+	case "pprof":
+		pprof := out.Tree.PprofStruct(&tree.PprofMetadata{
+			Type: out.SpyName, // TODO
+			Unit: out.Units,
+		})
+		out, err := proto.Marshal(pprof.Pprof())
+		if err == nil {
+			ctrl.writeResponseFile(w, "profile.pprof", out)
```

Comment:
I'm thinking it would be kind of nice to have a full name for a profile, for example it would be cool if it was something like:
```
<query> - <start time>.pprof
```
so for example:
```
pyroscope.server.cpu{} - 2021-11-11 15:00:00Z.pprof
```